src/Hydration.js

```diff
@@ -1,16 +1,55 @@
+import hydrationData from './data/hydration';
+
```

Comment:
Ok so I see and remember that we did this now, here we are importing hydrationData from from the data files that still exist in this repository. Currently this works for displaying information with the DOM because those files are still available. In scripts, they are commented out. I am not sure we can use the same methodology with the API promises.